### PR TITLE
store blog updates commit in branch `backup`

### PR DESCRIPTION
**Pull request by HuJJ-NB**
it can make branch `master` clear.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,20 +8,26 @@
...
-[RSS Feed](https://raw.githubusercontent.com/{repo_name}/master/feed.xml)
+MD_HEAD = """# HuJJ-NB
+My personal blog using issues and GitHub Actions
+Subscribe to [RSS Feed](https://raw.githubusercontent.com/{repo_name}/master/feed.xml)
+"""
+
+MD_FOOT = """
+---
+See also [https://imhujj.cc](https://imhujj.cc)
+Thanks for [yihong0618](https://github/com/yihong0618/gitblog)
```
Comment (yihong0618) [inline main.py:19]: do not add this~

File: main.py [hunk 2]
```diff
@@ -8,20 +8,26 @@
 from feedgen.feed import FeedGenerator
 from lxml.etree import CDATA
 
-MD_HEAD = """## Gitblog
-My personal blog using issues and GitHub Actions (随意转载，无需署名)
-[RSS Feed](https://raw.githubusercontent.com/{repo_name}/master/feed.xml)
+MD_HEAD = """# HuJJ-NB
+My personal blog using issues and GitHub Actions
```
Comment (yihong0618) [inline main.py:12]: keep old

File: main.py [hunk 3]
```diff
@@ -8,20 +8,26 @@
...
+
+MD_FOOT = """
+---
+See also [https://imhujj.cc](https://imhujj.cc)
+Thanks for [yihong0618](https://github/com/yihong0618/gitblog)
 """
 
 BACKUP_DIR = "BACKUP"
-ANCHOR_NUMBER = 5
+ANCHOR_NUMBER = 4
```
Comment (yihong0618) [inline main.py:23]: why change this?

File: main.py [hunk 4]
```diff
@@ -161,18 +167,19 @@ def add_md_firends(repo, md, me):
                     print(str(e))
                     pass
     with open(md, "a+", encoding="utf-8") as md:
-        md.write("## 友情链接\n")
+        md.write("\n## 友情链接\n\n")
         md.write(s)
+        md.write("\n> 通过向[友情链接](https://github.com/HuJJ-NB/HuJJ-NB/issues/6)评论的形式，将博客加入友链列表\n")
```
Comment (yihong0618) [inline main.py:172]: make it configable

File: main.py [hunk 5]
```diff
@@ -161,18 +167,19 @@ def add_md_firends(repo, md, me):
...
 def add_md_recent(repo, md, me, limit=5):
     count = 0
     with open(md, "a+", encoding="utf-8") as md:
         # one the issue that only one issue and delete (pyGitHub raise an exception)
         try:
-            md.write("## 最近更新\n")
+            md.write("\n## 最近更新\n\n")
             for issue in repo.get_issues():
-                if is_me(issue, me):
+                if issue.get_labels().totalCount & is_me(issue, me):
```
Comment (yihong0618) [inline main.py:182]: & -> and maybe more readable in Python

**Comment by yihong0618** [comment]
@HuJJ-NB 感谢，有些需要修改的更通用一点～

File: main.py (hunk 5 above)
Comment (HuJJ-NB) [reply main.py]: I deleted it.

**Comment by HuJJ-NB** [comment]
I try to test my `main.py` in my repo and it needs to make this branch default, that makes the branch looked be deleted. I'll give you another pr latter.

**Pull request closed without merging**